Source/UIView+Extensions.swift

```diff
@@ -11,4 +11,8 @@ extension UIView {
 
     return y
   }
+
+  var isiPhoneX: Bool {
+    return safeYCoordinate > 20
```

Comment:
What is `safeYCoordinate`? 🤔

Reply:
I think @zenangst named that 😬 But it's just the offset of the safe area insets

Reply:
Yeah it was based of this PR - https://github.com/hyperoslo/Whisper/pull/181/files

Basically this refactoring: https://github.com/hyperoslo/Whisper/pull/183/files